### ISSUE TITLE
Shell expansion to ../.. triggers on ... instead of ....

### ISSUE THREAD

**Issue by treuherz** [sorin-ionescu/prezto#1464]
According to the comments on the relevant function, this expansion should turn .... into ../.. . It triggers a period early on my install, which is a nightmare since I work a lot with bazel, for which you frequently have to type //... as an argument. I can't read shell glob syntax well enough to figure out if this is a bug in the implementation or an oddity of my install, though I'm not aware of having anything else runnigng that should affect this.

https://github.com/sorin-ionescu/prezto/blob/9f1a41de9015e89c5553289173e2e1a7e5a0b35d/modules/editor/init.zsh#L199

**Comment by belak** [comment]
Interesting...

If you want to disable it you can do something like https://github.com/sorin-ionescu/prezto/blob/9f1a41de9015e89c5553289173e2e1a7e5a0b35d/modules/editor/init.zsh#L372-L374 after sourcing init.zsh in your zshrc. This will let you revert to the default behavior or `.`

**Comment by treuherz** [comment]
Have already done similarly. Default behaviour is still not as documented, though!

**Comment by malikoth** [comment]
You're right, it is not behaving as documented.  I believe however, that this is an error with the documentation, and not the implementation.  The functionality is such that you would type `..` as normal, and then each addition press of `.` will add a `/..` to the end of the path.  Thus, three dots goes up two directories, four dots goes up three, etc.  Documentation should be updated to reflect this.
